### PR TITLE
Migrate publishing to repo.grdev.net

### PR DESCRIPTION
**Pull request by blindpirate**

### CODE REVIEW

**Comment by blindpirate** [comment]
@bot-gradle test QF

**Comment by blindpirate** [comment]
@bot-gradle test RFN

File: .teamcity/src/main/kotlin/common/extensions.kt [hunk 1]
```diff
@@ -125,6 +125,7 @@ fun BuildType.paramsForBuildToolBuild(buildJvm: Jvm = BuildToolBuildJvm, os: Os)
         param("env.GRADLE_OPTS", "-Xmx1536m -XX:MaxPermSize=384m")
         param("env.ANDROID_HOME", os.androidHome)
         param("env.ANDROID_SDK_ROOT", os.androidHome)
+        param("env.GRADLE_INTERNAL_REPO_URL", "%gradle.internal.repository.url%")
```
Comment (blindpirate) [inline .teamcity/src/main/kotlin/common/extensions.kt:128]: This uses environment variable intentionally so all forked process can inherit this value.

**Comment by blindpirate** [comment]
@bot-gradle test QF

**Review by wolfs — APPROVED** [review]
LGTM!

**Comment by blindpirate** [comment]
@bot-gradle test QF

**Comment by blindpirate** [comment]
The publication works: https://builds.gradle.org/viewLog.html?buildId=45162047&buildTypeId=Gradle_Master_Promotion_PublishBranchSnapshotFromQuickFeedback&tab=buildResultsDiv

**Comment by blindpirate** [comment]
@bot-gradle test and merge

**Comment by blindpirate** [comment]
@bot-gradle cancel

**Comment by wolfs** [comment]
@bot-gradle test and merge

**Comment by wolfs** [comment]
@bot-gradle test and merge

**Pull request merged**